### ISSUE TITLE
Example index.html fails with Uncaught ReferenceError: leafletPolycolor is not defined

### ISSUE THREAD

**Issue by bahco** [Oliv/leaflet-polycolor#16]
I've cloned `https://github.com/Oliv/leaflet-polycolor/` and symlinked `leaflet-polycolor/examples/index.html` and `leaflet-polycolor/dist/leaflet-polycolor.min.js` to a new directory `leaflet-polycolor` in my Apache2 Document root. Attempting to access that `leaflet-polycolor/index.html` in my browser results in an empty page and error message:

```
Uncaught ReferenceError: leafletPolycolor is not defined
    at HTMLDocument.<anonymous> ((index):20)
```

**Comment by Oliv** [comment]
Hi, a long time has passed since I used this module outside a webpack env, I will try to look into it soon

Regards, Oliv

**Comment by TrueWatcher** [comment]
Hi, I've got a similar problem. 

I'm trying to use leaflet-polycolor.min.js in a web page. I follow your example and put `<script src="./leaflet-polycolor.min.js"></script>` inside the page's < head > and `var leafletPolycolor = require('leaflet-polycolor');
  leafletPolycolor.default(L);` inside my script in its < body >.

 When I use the leaflet-polycolor.min.js from your example page (https://oliv.github.io/leaflet-polycolor/js/leaflet-polycolor.min.js) it basically works, but the same file from this repo's master branch fails with "Uncaught ReferenceError: require is not defined". Checked in Firefox 119 and Chromium 117.0.5938.149

Can your put a browser-workable script in the repo ? Or how about non-minified version of the workable old script from the example?
